### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by gabriel-maxx** [Cpasjuste/pplay#67]
hello I would like to know how to fix black bar in videos?

**Comment by Cpasjuste** [comment]
Hi,

I think if there is a problem with "black bars" it comes from your videos as this is internally handled by MPV and seems to works fine here.

**Comment by gabriel-maxx** [comment]
> Hi,
> 
> I think if there is a problem with "black bars" it comes from your videos as this is internally handled by MPV and seems to works fine here.

could you consider a feature in future versions to fix black bars like vlc or movian please ?

**Comment by Cpasjuste** [comment]
> > Hi,
> > I think if there is a problem with "black bars" it comes from your videos as this is internally handled by MPV and seems to works fine here.
> 
> could you consider a feature in future versions to fix black bars like vlc or movian please ?

Hi,

Could you share some pictures of the problem? I'm not sure what it is, maybe you want to change aspect ratio ?

**Comment by gabriel-maxx** [comment]
@Cpasjuste 
![pplay @ cpasjuste_20220219142226](https://user-images.githubusercontent.com/87019952/154849929-f8835c1a-e073-4f98-abc6-4cff0ef17130.jpg)

**Comment by Cpasjuste** [comment]
> @Cpasjuste ![pplay @ cpasjuste_20220219142226](https://user-images.githubusercontent.com/87019952/154849929-f8835c1a-e073-4f98-abc6-4cff0ef17130.jpg)

Thanks for the pic!

If you remove this black bars, the image will be stretched to a wrong aspect ratio, making all characters (this is more visible on them) taller and so they will look strange. I'm not sure you really want this !

**Comment by nikosama012** [comment]
hello, would it be possible to add an option that allows automatically playing all the videos inside a folder? That is, when a video is finished playing, the next video that is in the same folder is played automatically.

**Comment by gabriel-maxx** [comment]
> > @Cpasjuste ![pplay @ cpasjuste_20220219142226](https://user-images.githubusercontent.com/87019952/154849929-f8835c1a-e073-4f98-abc6-4cff0ef17130.jpg)
> 
> Thanks for the pic!
> 
> If you remove this black bars, the image will be stretched to a wrong aspect ratio, making all characters (this is more visible on them) taller and so they will look strange. I'm not sure you really want this !

I don't know which method you're going to say but I used two methods in movian I liked it and it always worked called "horizontal scale" and "vertical scale" and I didn't find it strange... is it possible to use this same method?

**Comment by Cpasjuste** [comment]
Yes this is something I can add in the future, but this is a sacrilege to me, especially on the movie in your screenshot :p

**Comment by gabriel-maxx** [comment]
@Cpasjuste so could you tell me what methods at the moment i could try to remove the black bar?

**Comment by Cpasjuste** [comment]
You can create a file named "mpv.conf" in "/data/pplay/mpv" (on PS4) and play with MPV options as seen here: https://mpv.io/manual/master/

Something like "keepaspect=no" (or maybe "no-keepaspect") could do the trick but I didn't test it myself.
